src/models/pang10/README.md

```diff
@@ -0,0 +1,47 @@
+# Pangolin (pang10)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Pangolin](./img/pang10.png)
+
+The System76 Pangolin is a laptop with the following specifications:
+
+- CPU
+    - Supports AMD Ryzen (3rd Gen) Mobile CPUs
+        - [Ryzen 5 4500U](https://www.amd.com/en/products/apu/amd-ryzen-5-4500u)
+        - [Ryzen 7 4700U](https://www.amd.com/en/products/apu/amd-ryzen-7-4700u)
+- Firmware
+    - System76 Firmware (non-open)
+- EC
+    - ITE IT8587
```

Comment:
Pages 1-8 (mainboard overview, top) and B-23 (schematic diagrams) of the upstream Clevo manual refer to the ITE IT5570. Is the 8587 what's actually in the machine?

Reply:
You're right, it's the 5570. I meant to circle back to that and look it up but forgot about it.